.travis.yml

```diff
@@ -1,9 +1,22 @@
 # Copyright 2018 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
 
-sudo: false
-language: node_js
-node_js:
-  - "8.3"
+
 notifications:
   disabled: true
+matrix:
+  include:
```

Comment:
By using a matrix, you are effective creating 3 jobs.  One for go, one for python, and one for node.js.

But if you want to do install work as part of the node.js job, you use the `before_install` or `install` keyword for that job. I see any reason to use a matrix yet.

https://docs.travis-ci.com/user/job-lifecycle/
https://docs.travis-ci.com/user/build-matrix/